documentation/source/building.rst

```diff
@@ -107,6 +107,10 @@ Environment Variables
     If defined, log lines displayed in terminal will be shorter. It will print only
     time, message type (``INFO``, ``WARNING``, ``ERROR``) and log message.
 
+.. envvar:: COCOTB_PDB_ON_EXCEPTION
+
+   If defined, cocotb will drop into the python debugger if a test fails with an exception.
```

Comment:
nit: python -> Python

Reply:
```suggestion
   If defined, cocotb will drop into the Python debugger (:mod:`pdb`) if a test fails with an exception.
```

Reply:
@imphil, does it let you hit the "commit suggestion" button above? It doesn't let me...

Reply:
no, he most likely disabled "maintainers can update this PR"

Reply:
Seems odd github would show me this then:

![image](https://user-images.githubusercontent.com/425260/72481191-9aae9480-37f1-11ea-9ae9-be8be201c887.png)

I've opened an issue with github support about this ambiguity

Reply:
Weird.  I dont't think I disabled that option, and I'm not seeing the box to enable it where it should be.